core/src/main.cpp

```diff
@@ -691,11 +692,11 @@ auto init_irods_connection_pool(const json& _config) -> std::unique_ptr<irods::c
 		opts);
 } // init_irods_connection_pool
 
-auto load_oidc_configuration(const json& _config, json& _oi_config, json& _endpoint_config) -> void
+auto load_oidc_configuration(const json& _config, json& _oi_config, json& _endpoint_config) -> bool
```

Comment:
Does this function need a top-level reference to `http_server` or can the `openid_connect` object be passed directly?

Reply:
Removed one parameter, as was no longer needed.